internal/controller/githubissue_controller.go

```
@@ -83,7 +92,9 @@ func (r *GithubIssueReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 			// Return and don't requeue
 			logStr := fmt.Sprintf("GithubIssue CR was not found, CR Name %s CR Namespace %s", req.Name, req.Namespace)
 			log.Info(logStr)
-			return emptyResult, nil
+			r.syncCRwithGitHub(ctx, accessToken)
```

golangci-lint (1.64.8)

Error return value of `r.syncCRwithGitHub` is not checked

(errcheck)